service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/spring/KeycloakServiceImpl.java

```diff
@@ -385,4 +385,23 @@ public String strip(String fqdn) {
 	public String seperator() {
 		return conf.getRooturlMatcher().seperator();
 	}
+
+    @Override
+    public boolean updateUserAttribute(String aliasId, UUID guid) {
+        if (!isInitialized()) {
+            throw new RuntimeException("keycloak not initialized");
+        }
+
+        UserRepresentation user = getUser(aliasId);
+        if (user == null) {
+            logger.info("Unable to find user {}, in keycloak", aliasId);
+            return false;
+        }
+
+        UsersResource users = this.realm.users();
+        UserResource userResource = users.get(user.getId());
+        user.singleAttribute(USER_ID_PROPERTY, guid.toString());
+        userResource.update(user);
+        return true;
```

Comment:
What if `userResource` is null ? i.e., `users.get(...)` did not find the specified ID

Reply:
it cannot be, since we check it in the condition before
`if (user == null) {
            logger.info("Unable to find user {}, in keycloak", aliasId);
            return false;
        }`